service-loadbalancer/gce/README.md

```diff
@@ -0,0 +1,18 @@
+# GCE Load Balancers
+
+
+## Disclaimer:
+- This is a **work in progress**.
+- For fault tolerant load balancing of ingress traffic, you need:
+  1. Multiple hosts running load balancers
```

Comment:
OK, so what do you need me to look at? :)

On Thu, Oct 1, 2015 at 10:40 PM, Prashanth B <notifications@github.com>
wrote:

> In service-loadbalancer/gce/README.md
> <https://github.com/kubernetes/contrib/pull/132#discussion_r40992742>:
>
> > @@ -0,0 +1,18 @@
> > +# GCE Load Balancers
> > +
> > +
> > +## Disclaimer:
> > +- This is a **work in progress**.
> > +- For fault tolerant load balancing of ingress traffic, you need:
> > +  1. Multiple hosts running load balancers
>
> heh, no i just copied that over from the other serviceloadbalancer README.
> I'm writing e2es atm, figured that would somehow validate this enough that
> you won't have to do some hardcore review.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/132/files#r40992742>.
>